io/src/pcd_grabber.cpp

```diff
@@ -246,7 +246,7 @@ pcl::PCDGrabberBase::PCDGrabberImpl::closeTARFile ()
   io::raw_close (tar_fd_);
   tar_fd_ = -1;
   tar_offset_ = 0;
-  memset (&tar_header_.file_name[0], 0, 512);
+  std::fill_n(tar_header_.file_name, 100, 0);
```

Comment:
Why the switch from 512 to 100?

Reply:
Because `char file_name[100];`

Uh-oh, is 512 the size of `tar_header_`? Yikes, will fix.